src/node-backend/adapters/database-adapter.js

```diff
@@ -1,135 +1,156 @@
 import {pool} from "./database-connection.js";
 
+export function createClassroom(username, classname) {
+    try {
+        const newClassroomQuery = "INSERT INTO `classrooms`(`name`,`teacherUsername`) VALUES (?, ?)";
```

Comment:
shouldn't we generate classId as well?

Reply:
yes, but it's generated automatically because the field in the database is set to AUTO_INCREMENT :) I return the classId so that frontend can use it afterwards to get the classroom info